app/helpers/work_packages_helper.rb

```diff
@@ -590,22 +597,29 @@ def work_package_form_assignable_versions_attribute(form, work_package, locals =
     return if work_package.assignable_versions.empty?
 
     version_field = work_package_form_field do
-      field = form.select(:fixed_version_id,
-                          version_options_for_select(work_package.assignable_versions,
-                                                     work_package.fixed_version),
-                          include_blank: true)
-
-      if authorize_for('versions', 'new')
-        field += prompt_to_remote(icon_wrapper('icon icon-add icon4',
-                                               I18n.t(:label_version_new)),
-                                  l(:label_version_new),
-                                  'version[name]',
-                                  project_versions_path(locals[:project]),
-                                  class: 'no-decoration-on-hover grid-block',
-                                  title: l(:label_version_new))
+      label = form.label(:fixed_version_id)
+
+      field = content_tag(:span, class: "form--field-container") do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.